### PR TITLE
Groups Controller edited and added EmailService

### PR DESCRIPTION
**Pull request by Bathande**

### CODE REVIEW

File: SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs [hunk 1]
```diff
@@ -13,6 +13,8 @@ namespace SkietbaanBE.Controllers
     [Route("api/Groups")]
     public class GroupsController : Controller
     {
+        EmailService mailS = new EmailService();
```
Comment (RashadAkoodieRabbit) [inline SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs]: please rename this variable to something like "mailService"

File: SkietbaanBE/SkietbaanBE/EmailService.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+namespace SkietbaanBE
+{
+    public class EmailService
+    {
+
+        readonly string smtpAddress = "smtp.gmail.com";
+        readonly int portNumber = 587;
+        readonly bool enableSSL = true;
+        readonly string emailFromAddress = "bathande704@gmail.com"; //Sender Email Address
+        readonly string password = "22842633"; //Sender Password
```
Comment (RashadAkoodieRabbit) [inline SkietbaanBE/SkietbaanBE/EmailService.cs]: Why are your passwords stored like this, I'm sure that has been discussed already

File: SkietbaanBE/SkietbaanBE/EmailService.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            {
+                SmtpClient client = new SmtpClient("smtp.gmail.com", 587);
+                client.EnableSsl = true;
+                client.DeliveryMethod = SmtpDeliveryMethod.Network;
+                client.Credentials = new NetworkCredential("bathande704@gmail.com", "22842633");
+                MailMessage msg = new MailMessage();
+                msg.To.Add(email);
+                msg.From = new MailAddress(emailFromAddress);
+                msg.Subject = "test";
+                msg.Body = "awe";
```
Comment (RashadAkoodieRabbit) [inline SkietbaanBE/SkietbaanBE/EmailService.cs]: This is awful! If you're committing this please put some better content

**Comment by RashadAkoodieRabbit** [comment]
Please address all whitespace issues, I'm not going to comment on each one

File: SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs [hunk 4]
```diff
@@ -122,13 +124,28 @@ private bool GroupExists(int id)
...
+
+
         [HttpPost]
         [Route("add")]
         public void AddListUsers([FromBody] List<User> users)
             
         {
-            int groupId = _context.Groups.ToArray().Length;
-            Group group = _context.Groups.Find(groupId);
+            //Group groupId =( _context.Groups.ToArray())[_context.Groups.ToArray().Length-1];
```
Comment (FredJEhlers) [inline SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs]: GC

File: SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs (hunk 1 above)
Comment (Bathande) [reply SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs]: i will take it out i'm no longer using the email

File: SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs (hunk 4 above)
Comment (Bathande) [reply SkietbaanBE/SkietbaanBE/Controllers/GroupsController.cs]: i have took it out

File: SkietbaanBE/SkietbaanBE/EmailService.cs (hunk 2 above)
Comment (CilliersN) [reply SkietbaanBE/SkietbaanBE/EmailService.cs]: wow

Comment (Bathande) [reply SkietbaanBE/SkietbaanBE/EmailService.cs]: i was not using that class forgot to delete it

File: SkietbaanBE/SkietbaanBE/EmailService.cs (hunk 3 above)
Comment (Bathande) [reply SkietbaanBE/SkietbaanBE/EmailService.cs]: i have deleted this class i was testing i didn't use it

**Pull request closed without merging**